test/classes/bradc/paramInClass/weirdParamInit3.chpl

```diff
@@ -4,7 +4,7 @@ class C {
 
 class D {
   var cc = new owned C(2);
-  var c: borrowed C = cc.borrow();
+  var c: borrowed C(?) = cc.borrow();
```

Comment:
It's not obvious to me how this PR's focus affected this test's behavior (which feels more like a #22745 change).  Your change doesn't make `D` any less generic, and it doesn't seem to involve a class hierarchy (?).

Reply:
Lydia set me straight that this is because the PR is based on a branch from the other PR.  Obvious in retrospect, and I'm not sure how I missed it in the OP (I did look! :) ).